Makefile

```diff
@@ -22,8 +22,7 @@ repo-full-name:
 	@echo ${REPO_FULL_NAME}
 
 compile:
-	@echo ${MAKEFILE_PATH}
```

Comment:
don't we need the print anymore?

Reply:
I don't think so, I'm not sure why it's there :D